### PR TITLE
Html dev

### PR DESCRIPTION
**Pull request by JAlbertins**
Iekšlapa

### CODE REVIEW

**Comment by reinisz** [comment]
svg definīcijas ir, bet iekodēti ir png.

**Comment by JAlbertins** [comment]
Labots.

File: js/custom.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   $(document).ready(function(){
 
-    $('.sky-part').css({minHeight: $(window).height()});
+    $('.sky-part:not(.sky-part-v2)').css({minHeight: $(window).height()});
```
Comment (JanisPlots) [inline js/custom.js]: Can be done with CSS3 '100vh', but anyway can't find selector for element.
Should't this be modified for resize event below as well?

File: inpage.html [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
...75d0-6564-424d-a79d-a5344763b314" x1="32" y1="63" x2="32" y2="43"/><circle class="5ba875d0-6564-424d-a79d-a5344763b314" cx="32" cy="5" r="4"/><path class="5ba875d0-6564-424d-a79d-a5344763b314" d="M32,53c2,1,5,2.24,5,5a5,5,0,1,1-9-3"/><path class="5ba875d0-6564-424d-a79d-a5344763b314" d="M32,53c-2-1-7-3-7-7,0-2.46,2-3.81,4-5"/><path class="5ba875d0-6564-424d-a79d-a5344763b314" d="M36,51a5.21,5.21,0,0,0,3-5c0-3.87-5-6-7-7"/><path class="5ba875d0-6564-424d-a79d-a5344763b314" d="M35,25c2,0,5,2,5,6,0,4.42-6,7-8,8-2-1-8-3.58-8-8,0-4,3-6,5-6"/><path class="4b5e0f8b-7237-4c8b-b762-0790343f8d20" d="M29,21H25C11,21,3,16,1,10c14,0,14-5,19-5a4.14,4.14,0,0,1,4,4v3a3,3,0,0,0,3,3h2"/><line class="5ba875d0-6564-424d-a79d-a5344763b314" x1="22" y1="21" x2="22" y2="17"/><line class="5ba875d0-6564-424d-a79d-a5344763b314" x1="18" y1="21" x2="18" y2="16"/><line class="5ba875d0-6564-424d-a79d-a5344763b314" x1="14" y1="20" x2="14" y2="15"/><line class="5ba875d0-6564-424d-a79d-a5344763b314" x1="10" y1="18" x2="10" y2="14"/><path class="4b5e0f8b-7237-4c8b-b762-0790343f8d20" d="M35,21h4c14,0,22-5,24-11C49,10,49,5,44,5a4.14,4.14,0,0,0-4,4v3a3,3,0,0,1-3,3H35"/><line class="5ba875d0-6564-424d-a79d-a5344763b314"
```
Comment (JanisPlots) [inline inpage.html]: Remove commented code.

File: css/styles.scss [hunk 3]
```diff
@@ -5,5 +5,22 @@
 
 @import "bootstrap/bootstrap",
         "partials/variables",
+        "vendor/selectBoxIt",
+
         "partials/mixins",
-          "partials/structure";
\ No newline at end of file
+
+          "partials/structure",
```
Comment (JanisPlots) [inline css/styles.scss]: Minor code formatting issue

File: css/styles.scss [hunk 4]
```diff
@@ -5,5 +5,22 @@
 
 @import "bootstrap/bootstrap",
```
Comment (JanisPlots) [inline css/styles.scss]: Import only used bootstrap components.

File: js/custom.js [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
   $(document).ready(function(){
 
-    $('.sky-part').css({minHeight: $(window).height()});
+    $('.sky-part:not(.sky-part-v2)').css({minHeight: $(window).height()});
 
 
     $(window).scroll(function(){
```
Comment (JanisPlots) [inline js/custom.js]: Remove debug code: console.log(bgOff);

File: inpage.html (hunk 2 above)
Comment (JanisPlots) [inline inpage.html]: Cleanup.

Comment (JanisPlots) [inline inpage.html]: Cleanup. var contentString = "Some content";

Comment (JanisPlots) [inline inpage.html]: Remove commented code.

**Review by JanisPlots — CHANGES_REQUESTED** [review]
Remove unused .js files

File: js/custom.js (hunk 1 above)
Comment (JAlbertins) [reply js/custom.js]: More than 20% users could not see this, if using vh.

**Review by JAlbertins — COMMENTED** [review]
Viss šķiet ok. Boostrap atsevišķas komponentes - izskatīšu. Man nav tiesību apstiprināt Merge.

**Review by JAlbertins — COMMENTED** [review]
Bootstrap components for Nav and grid.

File: js/custom.js (hunk 1 above)
Comment (JanisPlots) [reply js/custom.js]: Please read the whole comment

File: js/custom.js (hunk 5 above)
Comment (JanisPlots) [reply js/custom.js]: Has not been done

File: inpage.html (hunk 2 above)
Comment (JanisPlots) [reply inpage.html]: Has not been done. Please see 'File changes' of pull request this view didn't put the comment on right line.

Comment (JanisPlots) [reply inpage.html]: Has not been done. Please see 'File changes' of pull request this view didn't put the comment on right line.

Comment (JanisPlots) [reply inpage.html]: Has not been done. Please see 'File changes' of pull request this view didn't put the comment on right line.

**Comment by gatisj** [comment]
Es saprotu ka tur vēl paliek kaut kādi sīkumi ko vēlāk jālabo, bet tagad merojam iekšā lai turpinātu procesu.

**Pull request merged**